docs/contributor/telemetry-load-test/README.md

```diff
@@ -12,6 +12,9 @@ This document describes a reproducible test setup to determine the limits and KP
 - curl 8.4.x
 - jq 1.6
 
+## Test Script
+
+All test scenarios use a single test script [run-load-test.sh](assets/run-load-test.sh), which provides four parameters: '`-t` for test type supported values are `traces, metrics, metricagent`, `-n` for test name e.g. `0.92 Traces`,`-m` for multi pipeline scenarios, and `-b` for backpressure scenarios
```

Comment:
```suggestion
All test scenarios use a single test script [run-load-test.sh](assets/run-load-test.sh), which provides the following parameters:
- '`-t` for test type, supported values are `traces, metrics, metricagent`
- `-n` for test name, such as `0.92 Traces`
- `-m` for multi pipeline scenarios
- `-b` for backpressure scenarios
```

Reply:
This section updated, please review again